extension/chrome/elements/pgp_block.ts

```diff
@@ -313,12 +313,12 @@ Catch.try(async () => {
       } else if (typeof decoded.text !== 'undefined') {
         decryptedContent = decoded.text;
       }
+      if (decoded.subject) {
+        decryptedContent = getEncryptedSubjectText(decoded.subject, isHtml) + decryptedContent;
+      }
       for (const att of decoded.atts) {
-        if (att.treatAs() !== 'publicKey') {
-          renderableAtts.push(att);
-        } else {
+        att.treatAs() !== 'publicKey' ? renderableAtts.push(att) :
```

Comment:
I like the succinctness, however please try to avoid ternary operators when not working with the return value, eg:

```js
// not good, traditional `if` is better, even though it takes up more space
x ? doOne() : doTwo();

// good
return x ? doOne() : doTwo();

// good
const y = x ? doOne() : doTwo();

// good
doSomething(x ? one() : two());
```

I checked the style guide and it doesn't mention it. Sorry about that! Please feel free to updated with my samples above: https://github.com/FlowCrypt/flowcrypt-browser/wiki/FlowCrypt-TypeScript-Style-Guide

Reply:
It would be great to have a tslint for this, but I'm not sure there is one.

Reply:
oh, okay, got it. I'll try to not make this mistake again. 
Thanks!